### PR TITLE
chore: Extract an @agoric/eslint-config package

### PR DESCRIPTION
**Pull request by kriskowal**

### CODE REVIEW

File: packages/eslint-config/eslint-config.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
       {
         "devDependencies": [
           "rollup.config.js",
-          "test/**/*.js",
+          "*test/**/*.js",
```
Comment (michaelfig) [inline packages/eslint-config/eslint-config.json]: Was this intentional?

File: packages/eslint-config/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@agoric/eslint-config",
+  "version": "0.1.0",
+  "description": "Agoric style rules",
+  "private": true,
```
Comment (michaelfig) [inline packages/eslint-config/package.json:5]: Why private?

**Review by michaelfig — APPROVED** [review]
LGTM, just a few minor things.

File: packages/eslint-config/eslint-config.json (hunk 1 above)
Comment (kriskowal) [reply packages/eslint-config/eslint-config.json]: Aye. Probably should generalize to `*test*` too, to catch test262.

```
❯ find packages -name '*?test' -depth 2
packages/eventual-send/integration-test
```

File: packages/eslint-config/package.json (hunk 2 above)
Comment (kriskowal) [reply packages/eslint-config/package.json:5]: I’m going to duplicate this in `ses` and converge them, then probably publish from `ses`.

**Pull request merged**